src/MaterialDesignThemes.Wpf/Converters/TextFieldHintVisibilityConverter.cs

```diff
@@ -5,6 +5,9 @@ namespace MaterialDesignThemes.Wpf.Converters;
 
 public class TextFieldHintVisibilityConverter : IValueConverter
 {
+    public static readonly TextFieldHintVisibilityConverter Instance = new();
+    public static readonly TextFieldHintVisibilityConverter StringIsEmptyInstance = new() { IsEmptyValue = Visibility.Collapsed, IsNotEmptyValue = Visibility.Visible};
```

Comment:
Should the name of this be inverted?
```suggestion
    public static readonly TextFieldHintVisibilityConverter StringIsNotEmptyInstance = new() { IsEmptyValue = Visibility.Collapsed, IsNotEmptyValue = Visibility.Visible};
```

Reply:
Agreed, Personally I would prefer to name "Instance" "StringIsEmptyInstance", so we then have a "StringIsEmptyInstance" and a "StringIsNotEmptyInstance". Then it is not implicit anymore what the "Instance" version means.

Furthermore what do you think about removing the "String" part, since the converter already states that it is for a "TextField" implicitly meaning it is a "String". Therefore having a "IsEmptyInstance" and a "IsNotEmptyInstance" would make more sense.